Pillar_II/esm/README.md

```diff
@@ -156,6 +156,35 @@ The pruning task inspects each members data, dynamically, and modifies a column
 in the Cassandra database for that member. The next time FESOM2 looks at the column,
 it will understand it must stop.
 
+#### Running with containers
+
+To run with containers we have created `launch_fesom2_cont.sh` to launch all the workflow stack (PyCOMPSs, Hecuba and Fesom2) using a continer image.
```

Comment:
s/Fesom2/FESOM2 (for consistency with rest of doc)

s/continer/container

Can you also break the line somewhere near where the other lines are. We didn't set a limit of 80/120/etc, but tried to be consistent and to avoid very long lines - https://sembr.org/